### DISCUSSION TITLE
a flame graph for tracing view

### DISCUSSION THREAD

**Discussion by alburthoffman** [grafana/grafana#71807] [Observability]
At present, Grafana can view one trace as a tree and a graph, which are useful when few spans in the trace. But when there are lots of spans, like 100 or even more. The tree view and dependency graph are hard to explore.

A flame graph like https://www.brendangregg.com/flamegraphs.html might be helpful. It's good to allow user switch to use flame graph to display one trace.

the better part of the flame graph is when one span gets clicked, it can only show that span and underlying childs. It allows a different levels of granularity.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). To facilitate this, I will be closing this discussion.

Please check out the documentation on the flame graph we have in Grafana https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/flame-graph/

If this doesnt suit your needs, feel free to make an [issue](https://github.com/grafana/grafana/issues) to request the feature.
